multiclass/description-of-letter-recognition.txt

```diff
@@ -0,0 +1,62 @@
+1. Title: Letter Image Recognition Data 
```

Comment:
can you make "description" a suffix rather than a prefix, so the files are sorted together?